app/experimenter/experiments/models/nimbus.py

```diff
@@ -410,6 +410,9 @@ def bucket_namespace(self):
         if self.channel:
             keys.append(self.channel)
 
+        if self.is_rollout:
+            keys.append("rollout")
```

Comment:
Yes totally but I was thinking we should also do the other side and add `experiment` if it's not a rollout so you can quickly tell just by looking at it what it is.

Reply:
Oh and also I just realized we should wait on landing this until I get the fix for #6649 up because otherwise we'll have the same problem where the namespace will change midflight.

Reply:
I wanted to avoid that in order to not change all the namespaces everywhere

Reply:
Yeah but changing the buckets for inflight is fully a bug so we should just fix it first and then land this after with both keys instead of doing it in two parts, I'm wrapping up the fix now.

Reply:
I don't mean inflight so much as the shared bucket namespace between one live experiment and one launched after this. Before the change, they would have taken up buckets in the same namespace. After, they end up in different namespaces even though everything else is equal

Reply:
Ooohhhh yes.  You are right that is a very valid point.   Welllllll....we could either leave it out and either do some kind of migration and/or wait for some lull later, or just make the cutover now and deal with a bit of overlap until all the old experiments are done.  What do you think?

Reply:
Yeah nevermind what I said your original approach was correct, it's better to be as non disruptive as possible.  If we want to add `experiment` as a key we'll find a way to do it later that doesn't disrupt users workflows.